core/controllers/admin.py

```diff
@@ -871,6 +869,7 @@ def put(self):
             raise self.InvalidInputException(
                 'Invalid language_code: %s' % language_code)
 
+        removal_type = self.payload.get('removal_type')
```

Comment:
This is just some code style changes i.e, moving this line just before it's required!